arch/src/aarch64/mod.rs

```diff
@@ -200,6 +200,10 @@ pub fn initramfs_load_addr(
 static mut KERNEL_START: u64 = layout::KERNEL_START;
 static mut RAM_64BIT_START: u64 = layout::RAM_64BIT_START;
 static mut FDT_START: u64 = layout::FDT_START;
+static mut GIC_V3_DIST_START: u64 = layout::GIC_V3_DIST_START;
+static mut GIC_V3_REDISTS_ADDR: u64 = layout::GIC_V3_DIST_START - layout::GIC_V3_REDIST_SIZE;
+static mut GIC_V3_REDISTS_SIZE: u64 = layout::GIC_V3_REDIST_SIZE;
+static mut GIC_V3_ITS_ADDR: u64 = unsafe{GIC_V3_REDISTS_ADDR - layout::GIC_V3_ITS_SIZE};
```

Comment:
Not directly relate to this patch, but potentially we can have a `layout` structure that get initantiated according to some configurations (device tree, command line).